Makefile-legacy.rpmbuilder

```diff
@@ -48,6 +48,7 @@ endif
 dist-build-dep: refresh-update.$(YUM) dist-build-dep$(suffix $(PACKAGE))
 
 dist-build-dep.spec:
+	command -v rpmspec >/dev/null 2>&1
```

Comment:
Leftover debug?

Reply:
Rather, failed attempt at failing the build before missing-rpmspec-not-causing-failure :)